person.rb

```diff
@@ -0,0 +1,35 @@
+require './nameable'
+
+class Person < Nameable
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    super()
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
```

Comment:
- [ ] Please kindly include the relationship between Person and Rentals. A person can have many rentals, so we need a method that to add rentals. Please kindly implement that method.